sdk/component/conf.go

```diff
@@ -19,10 +19,14 @@ const (
 	errReasonCannotBeNil      = "cannot be nil"
 
 	// Env vars.
-	envVarKeyComponentName   = "SMITHY_COMPONENT_NAME"
-	envVarKeyWorkflowID      = "SMITHY_WORKFLOW_ID"
+	// -- BASE
+	envVarKeyComponentName = "SMITHY_COMPONENT_NAME"
+	envVarKeyInstanceID    = "SMITHY_INSTANCE_ID"
+	// -- LOGGING
 	envVarKeyLoggingLogLevel = "SMITHY_LOG_LEVEL"
-	envVarKeyBackedStoreType = "SMITHY_BACKEND_STORE_TYPE"
+	// -- STORE
+	envVarKeyBackendStoreType     = "SMITHY_BACKEND_STORE_TYPE"
```

Comment:
I think we should be able to infer the storage type from the store DSN. if it starts with 'sqlite' for example it will be SQLite while if starts with grpc it will be the remote implementation